API/Repositories/Implementations/OAuthClientRepository.cs

```diff
@@ -24,4 +20,20 @@ public async Task<bool> ValidateAsync(int clientId, string clientSecret)
 
         return match != null;
```

Comment:
```suggestion
        return client != null;
```
Refactor instances where `client` is referred to as `match`